server/src/test/java/org/opensearch/repositories/blobstore/BlobStoreRepositoryTests.java

```diff
@@ -265,10 +265,7 @@ public void testFsRepositoryCompressDeprecated() {
 
         new FsRepository(metadata, useCompressEnvironment, null, BlobStoreTestUtil.mockClusterService(), null);
 
-        assertWarnings(
-            "[repositories.fs.compress] setting was deprecated in OpenSearch and will be removed in a future release!"
-                + " See the breaking changes documentation for the next major version."
-        );
+        assertNoDeprecationWarnings();
```

Comment:
Because we can no longer support reading compress from env settings for FsRepo. Related thread: https://github.com/opensearch-project/OpenSearch/pull/9569#discussion_r1331555339